### DISCUSSION TITLE
NFTs from a blacklisted account aren't accessible to be listed/sold on the main node

### DISCUSSION THREAD

**Discussion by samiejimmons** [deso-protocol/dips#148] [Medium CIPs]
Buyers of NFTs from accounts that are later blacklisted are left with no way to access the NFT to relist/sell/move them; they're met with a 404 page and stuck with the NFT in their gallery. The only workaround I've found is to access them through a third party node.

I know burn/transfer options are coming next week (?) but it seems this will still be an issue - if the account is blacklisted I imagine it will behave the same and you won't be able to access the burn/transfer options either. 

Owners of a blacklisted creator's NFT should have access to the NFT, whether there should be stipulations on how it can be listed/accessed by other accounts (which shouldn't be an issue with burn/xfer options) or what they should be i don't know.  Having fraudulent/stolen NFTs simply locked in your gallery is not a good way for the blacklist function to handle it.

**Comment by Lux1n** [comment]
I agree completely. The same issue was happening a few months back with users being unable to view/sell CCs of blacklisted public keys on the bitclout node (ex. @HamillHimself). I believe that issue was resolved, so hopefully the same kind of fix can be implemented for NFTs?
